less/home-page.less

```diff
@@ -39,23 +96,66 @@
   }
 
   .content-destination {
-    width: 75%;
-    margin: 0 auto 30px;
+    width: 100%;
+    padding-top: 2.5rem;
+    padding-bottom: 2rem;
+    @media @desktop {
+      width: 75%;
+    }
 
     img {
       border-radius: 10px;
     }
   }
 
   .content-pick {
-    padding-top: 30px;
-    border-top: 2px dashed @navigation-border;
-    display: flex;
-    justify-content: space-between;
+    .flexy(flex, column, null, center, center);
+    padding: 3rem;
+    width: 92%;
+    border-top: 2px dashed gray;
+    @media @desktop {
+      width: 100%;
+      padding: 2rem 0;
+      .flexy(flex, row, null, space-between, center);
+    }
+    // padding-top: 30px;
+    // border-top: 2px dashed @navigation-border;
+    // display: flex;
+    // justify-content: space-between;
 
     .destination {
-      width: 30%;
-      margin-bottom: 30px;
+      width: 100%;
+      @media @desktop {
+        width: 30%;
+      }
+
+
+    }
+  }
+}
+.destination {
+  margin-bottom: 35px;
+  .btn {
+    .flexy(flex, row, null, center, center);
+    .btn-styles(#16A2B8, #fff, 100%, 4rem);
+    .center();
+    &:hover {
+      .pulse();
+      // animation-name: pulse;
+      // animation-duration: .2s;
+    }
```

Comment:
More of a cosmetic observation, but would you consider applying the pulsing animation to the `:active` pseudoclass instead (it might confuse people into thinking that they clicked on the button by accident)? Perhaps a non-pulsing scale-up animation would be more appropriate for the `:hover` pseudoclass.